crawler/emitter.py

```diff
@@ -241,6 +240,61 @@ def _publish_to_stdout(self):
                     print line.strip()
                     sys.stdout.flush()
 
+    def _publish_to_logstash(self, url, max_emit_retries=5):
```

Comment:
I kept it 'logstash' separate from 'http', in case anybody explicitly requires whole frame in a single blob over http, they can use http. 

i will add test-case for new emitter.